### PR TITLE
{tools}[intel/2018b] NCO v4.7.9, ANTLR v2.7.7

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2400.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2e13b0bce86c4461ce6a5e98e6a7c54b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3104.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/e95a5eb6e2bbe576ea691e68905eca1e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
node2060.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2f3652c926dcdaa06d06d298994ed61c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2015.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/fcb491f30785746ed8a3aae163385d15 for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 4 out of 4 (2 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/1f49234f9ee7d4c71a86151f05777b25 for a full test report.

File: easybuild/easyconfigs/a/ANTLR/ANTLR-2.7.7-intel-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+name = 'ANTLR'
+version = '2.7.7'
+
+homepage = 'http://www.antlr2.org/'
+description = """ANTLR, ANother Tool for Language Recognition, (formerly PCCTS)
+ is a language tool that provides a framework for constructing recognizers,
+ compilers, and translators from grammatical descriptions containing
+ Java, C#, C++, or Python actions."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
```
Comment (vanzod) [inline easybuild/easyconfigs/a/ANTLR/ANTLR-2.7.7-intel-2018b.eb]: Thoughts about moving it to `GCCcore`?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3132.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/7b86659731dd75411bf2cd3a261a1adf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2057.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/95422f16644cfb20fae574d32ccda3e4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2400.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/aed13ab238237c6908194c4a4c378d61 for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/0d7b7ec63fa13f0b0efcf2d597008e5f for a full test report.

**Comment by vanzod** [comment]
Going in, thanks @boegel!

**Comment by edmondac** [comment]
Test report by @bear-software
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
bear-pg0305u03a.bear.cluster - Linux RHEL 7.6, 8335-GTX, Python 2.7.5
See https://gist.github.com/2e60181a8022795242ac20694043add7 for a full test report.

**Comment by boegel** [comment]
@edmondac I presume that test report was submitted in error (it failed because `--robot` wasn't used)?

**Comment by edmondac** [comment]
Sorry @boegel  - I was aiming for 8136 not 8316!

**Pull request merged**